core/src/main/java/at/roteskreuz/stopcorona/skeleton/core/model/db/converters/DateTimeConverter.kt

```diff
@@ -7,17 +7,23 @@ import org.threeten.bp.ZoneId
 import org.threeten.bp.ZonedDateTime
 
 /**
- * Converter for dateTime using [ZonedDateTime] or [LocalDate].
+ * Converter for dateTime using [Instant], [ZonedDateTime] or [LocalDate].
  */
 class DateTimeConverter {
 
     @TypeConverter
     fun timestampToDateTime(timestamp: Long?): ZonedDateTime? =
-        timestamp?.let { ZonedDateTime.ofInstant(Instant.ofEpochSecond(it), ZoneId.of("UTC")) }
+        timestamp?.let { ZonedDateTime.ofInstant(Instant.ofEpochSecond(it), ZoneId.systemDefault()) }
```

Comment:
Once I'm thinking about this method, maybe the string representation would be better. Now the timezone is changed (time is correct, but it doesn't have to be in the same timezone as the source).

Not a bug, but if you would like it more precise, the refactoring is welcome.